pkg/sql/sem/tree/changefeed.go

```diff
@@ -10,9 +10,16 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgerror"
 )
 
+type ChangefeedLevel string
+
+const (
+	ChangefeedLevelDatabase ChangefeedLevel = "DATABASE"
```

Comment:
Instead of using strings, maybe this should be an enum where the zero value is `ChangefeedLevelTable`